kuma/users/models.py

```diff
@@ -188,6 +188,10 @@ class User(AbstractUser):
         blank=True,
         validators=[WEBSITE_VALIDATORS['discourse']],
     )
+    stripe_customer_id = models.CharField(
```

Comment:
Migrations are rare enough that I need to think through this, and when I think, I do it in a GitHub comment.

The deployment is done in Jenkins:

https://github.com/mozilla/kuma/blob/6472eed398cfce95be57ff18a5ce8372a8f155d1/Jenkinsfiles/push.groovy#L22-L28

1. The database is migrated while the old code is running. Django doesn't use `*`, but instead asks for each column by name. The old code won't load the new column.
2. The new code is rolled out. Users have a chance to get the new or old code. If the new code loads a user, it will include the (blank) customer ID.
3. The new code is rolled out, everything works.